app/views/words/edit.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Admins#edit</h1>
+<p>Find me in app/views/admins/edit.html.erb</p>
```

Comment:
use the same code in the updated words new file and make sure to use the partial `_form.html.erb` file